File: backend/src/main/java/com/voiz/dto/ReminderDto.java
package com.voiz.dto;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReminderDto {
	private String title;
	private String content;
	private LocalDate startDate;
	private LocalDate endDate;
}

File: frontend/app/src/main/java/com/voyz/datas/model/dto/ReminderDto.kt
package com.voyz.datas.model.dto

import java.time.LocalDate

/**
 * 리마인더 생성을 위한 DTO
 */
data class ReminderDto(
    val title: String,
    val content: String,
    val startDate: LocalDate,
    val endDate: LocalDate
) 